### PR TITLE
Fix linking analytics framework

### PR DESCRIPTION
**Pull request by faresbh**
**Problem**:
With the new `Analytics.framework `written in Swift, The HyBidDemo crashes on app start because of framework linking. 
The `Analytics.framework `was nested inside the `HyBid.xcframework `which is not supported in XCode. This will work perfectly fine, with cocoapods and solves all linking issues, then we will lose support for the drag and drop method. 

**Solution**:
I had to revert back the change and rewrite the framework in Objective-C.

### CODE REVIEW

File: PubnativeLite/PubnativeLite/Core/Anayltics/HyBidReportingEvent.m [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+#import "HyBidReporting.h"
+
+@implementation HyBidReportingEvent
+
+
+- (instancetype)initWith:(NSString *)eventType adFormat:(NSString *)adFormat properties:(NSDictionary<NSString *,NSString *> *)properties {
+    self = [super init];
+    if (self) {
+        self.eventType = eventType;
+        if (properties != nil) {
```
Comment (orkhanalizada2020) [inline PubnativeLite/PubnativeLite/Core/Anayltics/HyBidReportingEvent.m:33]: maybe you could use ternary operator here to shorten the lines. It is not critical, just an option

**Review by orkhanalizada2020 — APPROVED** [review]
Beside my comment - all looks good to me

File: PubnativeLite/PubnativeLite/Core/Anayltics/HyBidReportingEvent.m (hunk 1 above)
Comment (faresbh) [reply PubnativeLite/PubnativeLite/Core/Anayltics/HyBidReportingEvent.m:33]: definitely better

Comment (faresbh) [reply PubnativeLite/PubnativeLite/Core/Anayltics/HyBidReportingEvent.m:33]: done

**Pull request merged**